samples/apps/copilot-chat-app/webapi/DeploymentTemplates/sk-existing-azureopenai.bicep

```
@@ -6,9 +6,10 @@ Bicep template for deploying Semantic Kernel to Azure as a web app service with
 */
 
 @description('Name for the deployment')
```

Suggestion from Nilesh:

```suggestion
@description('Name for the deployment - Must consist of alphanumeric characters or \'-\'')
```